examples/bmp280/bmp280_example.c

```diff
@@ -6,66 +6,89 @@
 #include "FreeRTOS.h"
 #include "task.h"
 
+#include "i2c/i2c.h"
 #include "bmp280/bmp280.h"
 
 // In forced mode user initiate measurement each time.
 // In normal mode measurement is done continuously with specified standby time. 
 // #define MODE_FORCED
 
-const uint8_t scl_pin = 5;
-const uint8_t sda_pin = 4;
+const uint8_t scl_pin = 0;
+const uint8_t sda_pin = 2;
 
 #ifdef MODE_FORCED
 static void bmp280_task_forced(void *pvParameters)
 {
     bmp280_params_t  params;
-    float pressure, temperature;
+    float pressure, temperature, humidity;
 
     bmp280_init_default_params(&params);
     params.mode = BMP280_MODE_FORCED;
 
+    bmp280_calib_t bmp280_calib;
+    bmp280_calib.i2c_addr = BMP280_I2C_ADDRESS_0;
```

Comment:
Can rename this, was thinking the same. Perhaps some of the other drivers APIs could be harmonized too, such as the bmp180. Can we do that as a follow up as I want to get things working for a start.

Our data logger may well use the constants, log them at startup and then just log the ADC values to allow post calibration if necessary, but not certain yet. Could add an API to encapsulate access to them, but it just does not seem like the light weight style of this code base.